src/analysis/history.hpp

```diff
@@ -21,6 +21,12 @@
 #include <parthenon/package.hpp>
 #include <utils/error_checking.hpp>
 
+#include "geometry/geometry.hpp"
+#include "geometry/geometry_utils.hpp"
+#include "history_utils.hpp"
+#include "phoebus_utils/relativity_utils.hpp"
+#include "phoebus_utils/variables.hpp"
```

Comment:
Move as many of these include statements as possible out of the header file and into `analysis/history.hpp`. I suspect you can at least move `#include "geometry/geometry.hpp"` and `i#include "phoebus_utils/relativity_utils.hpp"` into `analysis/history.hpp`

Reply:
Done, thanks!